site/ja/docs/privacy-sandbox/topics/index.md

```diff
@@ -0,0 +1,392 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Topics API'
+subhead: >
+  Enable interest-based advertising, without having to resort to tracking the sites a user visits.
+description: >
+ A proposal for a mechanism to enable interest-based advertising without having to resort to tracking the sites a user visits.
+date: 2022-02-14
+updated: 2022-02-14
+authors:
+  - samdutton
+---
+
+## 実装ステータス
+
+このドキュメントでは、インタレスト ベース広告に関する新しい提案「Topics API」の概要を説明します。
+
+-  [Topics API
+   の提案](https://github.com/jkarlin/topics)は、[公開での検討段階](https://github.com/jkarlin/topics/issues)にあります。
+-  この提案についてフィードバックをお寄せください。ご協力いただける場合は、[「Topics API
+   の解説」リポジトリ](https://github.com/jkarlin/topics)で問題を作成するか、 
+   [「ウェブ広告ビジネスの改善」グループ](https://www.w3.org/community/web-adv/participants)で意見交換に参加してください。この解説には、さらなる定義が必要な[未解決の質問](https://github.com/jkarlin/topics/issues)が多く残っています。
+-  この API はまだどのブラウザにも実装されていません。
+-  [プライバシー サンドボックスのタイムライン](http://privacysandbox.com/timeline)では、Topics API や他のプライバシー
+   サンドボックスに関する提案の実装スケジュールをご確認いただけます。
+
+## この API が必要な理由
+
+Topics API は、ユーザーがアクセスしたサイトをトラッキングせずにインタレスト ベースの広告を掲載できるようにする、[プライバシー
+サンドボックス](http://localhost:8080/docs/privacy-sandbox/overview/)に関連した提案です。    
```

Comment:
@samdutton  This links to `localhost:8080`. The original article has this URL.

Reply:
Oops! Thanks. Fixed.

@devnook Might be worth checking for this in linting. (I think these localhost links can creep in when converting to/from Google Docs.)